pkg/indexmanagement/converters.go

```diff
@@ -0,0 +1,64 @@
+package indexmanagement
+
+import (
+	"fmt"
+	"strconv"
+
+	apis "github.com/openshift/elasticsearch-operator/pkg/apis/logging/v1"
+	"github.com/openshift/elasticsearch-operator/pkg/logger"
+)
+
+func calculateConditions(policy apis.IndexManagementPolicySpec, primaryShards int32) rolloverConditions {
+	// 40GB = 40960 1M messages
+	maxDoc := 40960 * primaryShards
+	// TODO add back in for es6
+	// maxSize := defaultShardSize * primaryShards
+	maxAge := ""
+	if policy.Phases.Hot != nil {
+		maxAge = string(policy.Phases.Hot.Actions.Rollover.MaxAge)
```

Comment:
`Rollover` field is also a pointer, so this code may still crash with `nil pointer` error